examples/dockerfiles/Kogito.dockerfile

```diff
@@ -1,30 +1,25 @@
-#Not yet released Change the 31 st oct with quay.io/repository/kiegroup/kogito-swf-builder:latest
-FROM quay.io/kiegroup/kogito-swf-builder-nightly:latest AS builder
+FROM quay.io/ricardozanini/kogito-swf-builder:latest AS builder
 
-# Kogito User
-USER 1001
+ARG QUARKUS_PACKAGE_TYPE="jar"
+ARG SCRIPT_DEBUG="false"
 
-# User home from base image
-WORKDIR /home/kogito/kogito-sw-base
-
-# Copy from build context to skeleton resources project
-COPY * ./src/main/resources/
-
-# Maven vars enhirited from the base image
-RUN ${MAVEN_HOME}/bin/mvn -U -B ${MAVEN_ARGS_APPEND} -s ${MAVEN_SETTINGS_PATH} clean install -DskipTests
+ # Copy from build context to skeleton resources project
+COPY * ./resources/
 
+RUN /home/kogito/launch/build-app.sh ./resources
 #=============================
 # Runtime Run
 #=============================
-FROM registry.access.redhat.com/ubi8/openjdk-11:1.11
+FROM registry.access.redhat.com/ubi8/openjdk-11:latest
```

Comment:
isn't better to use our kogito-runtime-jvm?

Reply:
Nops, why is that one better other than the official RH's one used by Quarkus?

Reply:
size, IIRC this is the openjdk-devel while we would need just the headless.
and we could customize if needed in the future, (and another reason to keep it around :) )

Reply:
Not sure if this is the way to go since it's the one used by Quarkus, and we could encounter issues in the long run. I'm open to a case study to understand if we can move. I'm skeptical, though. I wouldn't change it, honestly. Unless we have a strong reason not to, I'd rather stay with the one defined by Quarkus.

Reply:
imho size is a string reason :)
we can switch then to openjdk11-runtime
openjdk11: runtime x devel
                   354mb     391mb

kogito-runtime: 355mb
it is something to think about then :)
openjdk-runtime x openjdk devel

Reply:
yes, size is a strong reason if ours were like 20% less at least? Theirs is 354 and ours 355? Is that it? I don't see a reason then.

Reply:
Also, theirs is maintained by the JDK team.

Reply:
size difference 354mb x 391mb

Reply:
oh ok, you're saying we are using devel whilst they have a runtime one? not sure if I'm following.

Reply:
yes, they have
registry.access.redhat.com/ubi8/openjdk-11-runtime